src/utils/getIssueAntdPath.ts

```diff
@@ -17,21 +17,37 @@ const serializePath = (
   const baseSchema = getSchemaBaseSchema(schema);
   if (isZodObject(baseSchema)) {
     antdPath.push(subPath);
-    const subPathBaseSchema = getSchemaBaseSchema(baseSchema.shape[subPath]);
 
-    return isZodObject(subPathBaseSchema)
-      ? serializePath(subPathBaseSchema, zodPath, antdPath)
-      : concatAntdPath(antdPath);
+    const replacedPath = subPath.replace(/\.\d+/, "");
+    const subPathBaseSchema = getSchemaBaseSchema(
+      baseSchema.shape[replacedPath]
+    );
+
+    if (isZodObject(subPathBaseSchema))
+      return serializePath(subPathBaseSchema, zodPath, antdPath);
+    if (isZodArray(subPathBaseSchema))
+      return serializePath(subPathBaseSchema.element, zodPath, antdPath);
+
+    return concatAntdPath(antdPath);
   }
 
   throw new Error("Unknown zod schema");
 };
 
 const getIssueAntdPath = (schema: ZodTypeAny, issue: ZodIssue) => {
-  const zodPath = issue.path.reverse() as string[];
+  // issue.path = ["testArray", "0", "myField"] => modifiedPath = ["testArray.0", "myField"]
```

Comment:
Please remove comment leftover

Reply:
I just left it to make it clear how list paths had to be taken in order to work, would you still want me to remove it?

Reply:
It's not needed, we'll do it in another way.